index.html

```diff
@@ -0,0 +1,63 @@
+<html lang="en">
```

Comment:
- [ ] Kindly add the `DOCTYPE` declaration at the first line of your HTML document because it tells the browser what version of HTML the document is written in.